static/js/pages/instances/actions/StartStopInstanceBtn.tsx

```diff
@@ -56,7 +56,7 @@ const StartStopInstanceBtn: FC<Props> = ({
   const canBeStarted =
     instance.status === "Stopped" || instance.status === "Frozen";
   const canBeStopped =
-    instance.status === "Running" ||
+    instance.status.endsWith("ing") ||
```

Comment:
This approach might be working, but it's not straightforward for other developers. I would created a variable instead, like:
```
const startStatuses = ["Stopped", "Frozen"];
const stopStatuses = ["Running", ...];

const isDisabled = [...startStatuses, ...stopStatuses].includes(instance.status); // Or just pass that directly to `isDisabled` prop
```

Reply:
Good idea, adjusted to this.